Cargo.toml

```diff
@@ -1,15 +1,33 @@
+[workspace]
+members = ["extensions/*"]
+
+[workspace.package]
+authors = ["Nitrokey GmbH <info@nitrokey.com>"]
+edition = "2021"
+repository = "https://github.com/trussed-dev/trussed-staging"
+license = "Apache-2.0 OR MIT"
+
 [package]
 name = "trussed-se050-backend"
 version = "0.2.0"
-edition = "2021"
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+license.workspace = true
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[workspace.dependencies]
+serde = { version = "1.0.185", default-features = false, features = ["derive"] }
+trussed = { version = "0.1.0", features = ["serde-extensions"] }
 
 [dependencies]
+serde.workspace = true
+trussed.workspace = true
```

Comment:
I'm not sure I see the benefits of using workspace dependencies. This seems like added syntax for not much benefits no?

Reply:
The idea is that we only have to define the default features and the version number in one place, so bumping to Trussed 0.2.0 would only need a single change.  But I’m also fine with defining the dependencies in all extensions.